### DISCUSSION TITLE
Explain Custom Menu Locations better

### DISCUSSION THREAD

**Discussion by sc0ttkclark** [pods-framework/pods#6830] [Q&A]

**Comment by NikV** [comment]
Do we have this done in some form of documentation on Pods.io?

**Comment by jimtrue** [comment]
I think this may be related to this forum post: https://wordpress.org/support/topic/how-to-use-custom-menu-location?replies=2

This is for the help text added to Custom Menu Locations. Which is referenced under the WordPress Codex here: https://codex.wordpress.org/Function_Reference/add_submenu_page

**Comment by Shelob9** [comment]
I think we do not need to be in the business of restating the WordPress codex. If anything we (specifically @NikV) can add the relevant codex link to a help bubble.
